src/components/Layouts/SidePanelsLayout.tsx

```diff
@@ -0,0 +1,123 @@
+import { FC, ReactElement } from 'react';
+import { Box, styled, Theme } from '@material-ui/core';
+import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
+
+const FullHeightBox = styled(Box)(({ theme }) => ({
+  height: `calc(100vh - ${theme.mixins.toolbar.minHeight}px)`,
+  ['@media (min-width:0px) and (orientation: landscape)']: {
+    height: `calc(100vh - ${
+      (theme.mixins.toolbar[
+        '@media (min-width:0px) and (orientation: landscape)'
+      ] as { minHeight: number }).minHeight
+    }px)`,
+  },
+  ['@media (min-width:600px)']: {
+    height: `calc(100vh - ${
+      (theme.mixins.toolbar['@media (min-width:600px)'] as {
+        minHeight: number;
+      }).minHeight
+    }px)`,
+  },
+}));
+
+const ScrollBox = styled(FullHeightBox)({
+  overflowY: 'auto',
+});
+
+const CollapsibleWrapper = styled(Box)({
+  display: 'flex',
+  overflowX: 'hidden',
+});
+
+const ExpandingContent = styled(Box)(
+  ({ open }: { theme: Theme; open: boolean }) => ({
+    flexGrow: 1,
+    flexShrink: 0,
+    flexBasis: open ? 0 : '100%',
+    transition: 'flex-basis ease-in-out 225ms',
+    overflowX: 'hidden',
+  }),
+);
+
+const PanelWrapper = styled(ScrollBox)(
+  ({
+    theme,
+    breakpoint,
+  }: {
+    theme: Theme;
+    breakpoint: Breakpoint | number;
+  }) => ({
+    flexGrow: 0,
+    flexShrink: 0,
+    [theme.breakpoints.down(breakpoint)]: {
+      transition: 'transform ease-in-out 225ms',
+      background: theme.palette.common.white,
+      position: 'absolute',
+      zIndex: 1,
+    },
+  }),
+);
+
+const LeftPanelWrapper = styled(PanelWrapper)(
+  ({ theme, open }: { theme: Theme; open: boolean }) => ({
+    borderRight: `1px solid ${theme.palette.cruGrayLight.main}`,
```

Comment:
Is there a difference between this gray and ```theme.palette.grey```? ```theme.palette.grey``` has a lot of options as well so I was curious if there was something specific about ```theme.palette.cruGrayLight.main```.

Reply:
#EBECEC (`cruGrayLight`) is a custom color from Cru's brand. It isn't in `theme.palette.grey`https://material-ui.com/customization/default-theme/.

Maybe there's an argument that we should customize the grey palette but it's got 14 greys in it. I'm not sure how you'd know which one to pick and design isn't customizing them.